<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Spectrum</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_input"/>
   </item>
   <item>
    <widget class="LevelMeterWidget" name="widget_input_level" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>8</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveWidget" name="widget_wave" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LevelMeterWidget</class>
   <extends>QWidget</extends>
   <header>LevelMeterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WaveWidget</class>
   <extends>QWidget</extends>
   <header>WaveWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
